[CI] DocsClientYamlTestSuiteIT.test {yaml=reference\ilm\apis/get-lifecycle/line_49} failure

Failed two time today:
https://gradle-enterprise.elastic.co/s/42ybbarn5o7s6 
(https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+7.6+multijob-windows-compatibility/os=windows-2019/28/console)

https://gradle-enterprise.elastic.co/s/s6olxrjh67w7i 
(https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+7.6+multijob-windows-compatibility/os=windows-2012-r2/28/console)

Failure:
```
java.lang.AssertionError: Failure at [reference\ilm\apis/get-lifecycle:42]: $body didn't match expected value:
                         $body: 
                       my_policy: 
                     modified_date: same [2020-01-29T11:02:12.047Z]
                            policy: 
                              phases: 
                                delete: 
                                 actions: 
                                    delete: same [empty map]
                                 min_age: same [30d]
                                  warm: 
                                 actions: 
                                forcemerge: 
                            max_num_segments: same [1]
                                 min_age: same [10d]
                           version: expected Integer [1] but was Integer [3]

	at __randomizedtesting.SeedInfo.seed([16DA10F525639750:9E8E2F2F8B9FFAA8]:0)
	at org.elasticsearch.test.rest.yaml.ESClientYamlSuiteTestCase.executeSection(ESClientYamlSuiteTestCase.java:405)
	at org.elasticsearch.test.rest.yaml.ESClientYamlSuiteTestCase.test(ESClientYamlSuiteTestCase.java:382)
```

Cannot reproduce locally so far, maybe an issue just happening on Windows?
```
gradlew ':docs:integTestRunner' --tests "org.elasticsearch.smoketest.DocsClientYamlTestSuiteIT.test {yaml=reference\ilm\apis/get-lifecycle/line_49}" \
  -Dtests.seed=16DA10F525639750 \
  -Dtests.security.manager=true \
  -Dtests.locale=es-PY \
  -Dtests.timezone=America/Catamarca \
  -Dcompiler.java=13 \
  -Druntime.java=8
```

--- a/docs/src/test/java/org/elasticsearch/smoketest/DocsClientYamlTestSuiteIT.java
+++ b/docs/src/test/java/org/elasticsearch/smoketest/DocsClientYamlTestSuiteIT.java
@@ -143,14 +143,14 @@ public class DocsClientYamlTestSuiteIT extends ESClientYamlSuiteTestCase {
 
     protected boolean isSLMTest() {
         String testName = getTestName();
-        return testName != null && (testName.contains("/slm/") || testName.contains("\\slm\\") ||
+        return testName != null && (testName.contains("/slm/") || testName.contains("\\slm\\") || (testName.contains("\\slm/")) ||
             // TODO: Remove after backport of https://github.com/elastic/elasticsearch/pull/48705 which moves SLM docs to correct folder
-            testName.contains("/ilm/") || testName.contains("\\ilm\\"));
+            testName.contains("/ilm/") || testName.contains("\\ilm\\") || testName.contains("\\ilm/"));
     }
 
     protected boolean isILMTest() {
         String testName = getTestName();
-        return testName != null && (testName.contains("/ilm/") || testName.contains("\\ilm\\"));
+        return testName != null && (testName.contains("/ilm/") || testName.contains("\\ilm\\") || testName.contains("\\ilm/"));
     }
 
     protected boolean isMachineLearningTest() {

